cheerlights/app.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import . "github.com/alexellis/blinkt_go"
```

Comment:
Should we follow the principle used with progress and store the sysfs examples in the sysfs folder?

Also just spotted that I need to change the redis_view file to the new brightness regime